test/GitcoinGovernor.t.sol

```diff
@@ -427,3 +428,466 @@ contract GitcoinGovernorAlphaPostProposalTest is GitcoinGovernorProposalTestHelp
     governorAlpha.queue(_newProposalId);
   }
 }
+
+contract NewGitcoinGovernorProposalTest is GitcoinGovernorProposalTestHelper {
+  // From GovernorCountingSimple
+  uint8 constant AGAINST = 0;
+  uint8 constant FOR = 1;
+  uint8 constant ABSTAIN = 2;
```

Comment:
Should we instead define the same `enum VoteType` and use e.g. `VoteType.Against` instead?

Reply:
Ben added these, but I think I prefer it this way. I think it's clear and you don't have to cast the enum to uint8 all over the place

Reply:
I agree